test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/schema.py

```diff
@@ -523,7 +523,7 @@ def doc_schema(module_name, for_collection=False, deprecated_module=False):
         All(
             Schema(
                 doc_schema_dict,
-                extra=PREVENT_EXTRA
+                extra=ALLOW_EXTRA
```

Comment:
I'm not sure what this change means in terms of testing module docs against the schema. I like the new attributes. I'd also like to see them tested to prevent typos and other errors of omission and commission. It would be nice to avoid things like `asynch`/`async`.

Reply:
this is what I got from felix when asking how to add to the schema.